TestProjects/Playground/Playground.iOS/Playground.iOS.csproj

```diff
@@ -84,10 +84,10 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Assets.xcassets\AppIcon.appiconset\Contents.json">
-      <InProject>false</InProject>
```

Comment:
have you found out what these changes do then as they're the same ones that you didn't want to go in last time

Reply:
lol yeh good point - I just realised I missed them too
I had to adjust references in the project file, so missed this change

Reply:
Seems to be a VS thing - changes InProject to Visible element